test/fail/ok/implicit-import-hint-dot.tc.ok

```diff
@@ -0,0 +1,18 @@
+implicit-import-hint-dot.mo:13.3-13.12: type error [M0098], cannot implicitly instantiate function of type
+  <K, V>(map : Map<K, V>, compare : (implicit : (K, K) -> Order), key : K) ->
+    ?V
+to argument of type
+  (compare : (implicit : (K, K) -> Order), key : K)
+to produce result of type
+  ()
+because no instantiation of K, V makes
+  ?V  <:  ()
+and
+  {var root : Node<Text, Text>; var size_ : Nat}  <:  (map : Map<K, V>)
```

Comment:
I want to address this confusing error in https://github.com/dfinity/motoko/issues/5537